### ISSUE TITLE
Egress status stay active when participant disconnect

### ISSUE THREAD

**Issue by vincentbrison** [livekit/egress#143]
Hey, thanks for the awesome work. We are using LiveKit and it's egress system in production in GCP + k8s. We do live transcription through audio track egress, and we noticed something strange but not impacting regarding egress status. When a participant is leaving, the egress status stay `ACTIVE` (the result of the command `livekit-cli list-egress`) but we observe the correspond WebSocket to be close. Should not the status be `COMPLETE` instead?

We have a bash script to recreate this issue:

<details>
<summary>egress.sh</summary>

```bash
#!/usr/bin/env bash

###
# Script to load test livekit egress. It will start as much audio egress as nb_participants.
#
# Execute it like ./livekit-egress-loadtester.sh <livekit_url> <api_key> <api_secret> <nb_participants> <room_duration>
# Ex:
# ./loadtester.sh api_key api_secret 60 10m
#
# Params:
# livekit_url: The url of the livekit instance to be tested
# api_key: Livekit api key
# api_secret: Livekit api secret
# nb_participants: How many participants
# room_duration: How long room will last
###

TEST_ID=$(date -u +"%Y-%m-%dT%H_%M_%SZ")

LIVEKIT_URL=$1
API_KEY=$2
API_SECRET=$3
NB_PARTICIPANTS=${4-4}
TEST_TIME=${5-30s}

roomName="room-egress-test"
logFile="room-egress-test_$TEST_ID.log"

livekit-cli load-test \
  --url "$LIVEKIT_URL" \
  --api-key "$API_KEY" \
  --api-secret "$API_SECRET" \
  --room $roomName \
  --duration "$TEST_TIME" \
  --subscribers 1 \
  --publishers "$NB_PARTICIPANTS" \
  --audio-bitrate 100000 \
   | tee "$logFile" | sed -un "s/^subscribed to track .* \(.*\) audio.*/\1/p" | while read -r id; # Extract audio track ids to start egress
do
  sleep 1 # Avoid starting all egress at the same time, to avoid egress service to refuse to start because too much requests at the same time.
  echo "Starting egress for $id"
  requestFileName="request-$id.json"
  # Edit this line to change target of egress (for instance WS instead of bucket)
  echo "{ \"room_name\": \"$roomName\", \"track_id\": \"$id\", \"file\": { \"filepath\": \"livekit-egress-test/$TEST_ID/$id.ogg\" } }" > "$requestFileName"
  livekit-cli start-track-egress --url "$LIVEKIT_URL" --api-key "$API_KEY" --api-secret "$API_SECRET" --request "$requestFileName" &
done &

echo "egress room started."

loadTestPid=$!
wait $loadTestPid

echo "egress room ended."
```

</details>

By running this script, as many audio egress as participants will be started, and they will store their results on cloud storage. At the end of the test, we can find all the audio files, but if we run `livekit-cli list-egress`, most of the egress will still be in `ACTIVE` state.

Is is normal the status of the egress stay `ACTIVE` even if the corresponding participant leaved the room? Thanks!

**Comment by frostbyte73** [comment]
are you using v1.4.5? This was fixed yesterday in #140

**Comment by vincentbrison** [comment]
We are using egress 1.4.0 and LiveKit 1.2.1

**Comment by frostbyte73** [comment]
closing as fixed in 1.4.5
